core/integration/module_test.go

```diff
@@ -5183,6 +5183,103 @@ func (t *Test) IgnoreDirButKeepFileInSubdir(
 	})
 }
 
+func (ModuleSuite) TestFloat(ctx context.Context, t *testctx.T) {
```

Comment:
Hm, it feels weird to have a dedicated test for this - I was honestly expecting that we would have a more generic test where we pass a lot of different types in (e.g. int, string, etc) - but it doesn't seem like this exists?

Maybe I've missed it, but if it does, that would be a better place to add it (otherwise, up to you if you want to add it).

Reply:
I think I could add that in a follow up PR but not in that one